website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,412 @@
+---
```

Comment:
[nit] can we change the file name? I feel that a link like `/path/to/0.74-yoga-improvements-preparing-for-the-new-architecture` is a bit verbose. The `<release_date>.0.74` bit is already enough to characterize the file. This will produce easier anchors.

Reply:
I'm matching the slug format convention of our existing posts. We could shorten this to `0.74-announcement`.

The anchors are definitely fiddly 😅.